openedx_webhooks/webhook_confs.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Configuration for webhooks we want to install.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from flask import url_for
+
+
+WEBHOOK_CONFS = ({
+    'config': {
+        'url': url_for("github_views.pull_request", _external=True),
+        'content_type': 'json',
+        'insecure_ssl': False,
+    },
+    'events': ['pull_request']
+})
+"""
+Tuple[Dict[
+    str, Union[Dict[str, str], List[str]]
```

Comment:
`Dict[str, str]` here doesn't capture the type of what you have above. Two of your keys map to strings, but one is a boolean.  I'm not sure this type syntax is helping us much. Better would be to describe the keys in the dict, what they mean, and what value they expect.